pulsar-client-cpp/include/pulsar/Consumer.h

```diff
@@ -390,6 +390,17 @@ class PULSAR_PUBLIC Consumer {
      */
     bool isConnected() const;
 
+    /**
+     * Asynchronously get an ID of the last available message or a message ID with -1 as an entryId if the
+     * topic is empty.
+     */
+    void getLastMessageIdAsync(pulsar::GetLastMessageIdCallback callback);
```

Comment:
```suggestion
    void getLastMessageIdAsync(GetLastMessageIdCallback callback);
```

This method definition is already in namespace pulsar.